doc/introduction.rst

```diff
@@ -7,12 +7,78 @@ Introduction: nistats in a nutshell
+           Nistats is a python module to perform voxel-wise analyses of functional magnetic resonance images (fMRI) using linear models. It provides functions to create design matrices, at the subject and group levels, to estimate them from images series and to compute statistical maps (contrasts). It allows to perform the same statistical analyses than SPM or FSL (but it does not provide tools for preprocessing stages (realignement, spatial normalization, etc.); for this, see nipype???).
+
+
+
+A primer on BOLD-fMRI data analysis
+===================================
+
+Functional magnetic resonance imaging (fMRI) is based on the fact that when local neural activity increases, increases in metabolism and blood flow lead to fluctuations of the relative concentrations of oxyhemoglobine (the red cells in the blood that carry oxygen) and deoxyhemoglobine (the same red cells after they have delivered the oxygen). Because oxy and deoxy hemoglobine have different magnetic properties (one is diamagnetic while the other is paramagnetic), they affect the local magnetic field in different ways. The signal picked up by the MRI scanner is sensitive to these modifications of the local magnetic field. To record cerebral activity, during functional sessions, the scanner is tuned to detect this "Blood Oxygen Level Dependent" (BOLD) contrast.
+
+Brain activity is measured in sessions that last several minutes, during which the participant performs some kind of cognitive task while the scanner acquires brain images, typically every 2 or 3 seconds (the time between two successive image acquisition is called the Time of Repetition, or TR).
+
+A cerebral MR image provides a 3D image of the brain that can be decomposed in voxels (the equivalent of pixels, but in 3 dimensions).
+
+ INSERT HERE A PIC OF A BRAIN WITH A VOXEL GRID SUPERIMPOSED
+
+A typical step in the preprocessing of MR images, involves spatially morphing these images onto a standard template (e.g. the MNI152 template from the Montreal Neurological Institute). One this is done, the coordinates of a voxel are in the same space as the template and can be used to estimate its brain location using brain atlases.
+
+The series of images acquired during a functional session provide, in each voxel, a time series, sampled at the TR.
+
+INSERT HERE A SAMPLE OF A TIME SERIES IN A VOXEL (or several voxels)
+
+One way to analyze such times series consists in comparing them to a *model* buld from our knowledge of the events that occurred during the functional session. Events can correspond to actions of the participant (e.g. button presses), presentations of sensory stimui (e.g. sound, images), or hypothesized internal processes (e.g. memorization of a stimulus), ...
```

Comment:
built

Reply:
Ok to everything. I am correcting the typos and will work on the first tutorial (single subject/single session/bloc design)